ci/cloudbuild/dockerfiles/demo-opensuse-leap.Dockerfile

```diff
@@ -50,6 +49,103 @@ ENV PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:/usr/local/lib64/pkgconfig
 ENV PATH=/usr/local/bin:${PATH}
 # ```
 
+# #### Abseil
+
+# We need a recent version of Abseil. Enabling `ABSL_PROPAGATE_CXX_STD` 
+# propagates the version of C++ used to compile Abseil to anything that depends
+# on Abseil.
+
+# ```bash
+WORKDIR /var/tmp/build/abseil-cpp
+RUN curl -fsSL https://github.com/abseil/abseil-cpp/archive/20240116.1.tar.gz | \
+    tar -xzf - --strip-components=1 && \
+    cmake \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DABSL_BUILD_TESTING=OFF \
+      -DABSL_PROPAGATE_CXX_STD=ON \
+      -DBUILD_SHARED_LIBS=yes \
+      -S . -B cmake-out && \
+    cmake --build cmake-out -- -j ${NCPU:-4} && \
+    cmake --build cmake-out --target install -- -j ${NCPU:-4} && \
+    ldconfig
+# ```
+
+# #### RE2
```

Comment:
I think you can continue using RE2 from the packages.

Reply:
My testing says no:

```
/usr/lib64/gcc/x86_64-suse-linux/8/../../../../x86_64-suse-linux/bin/ld: /usr/local/lib/libgrpc.so.39.0.0: undefined reference to `re2::RE2::RE2(absl::lts_20240116::string_view, re2::RE2::Options const&)'
/usr/lib64/gcc/x86_64-suse-linux/8/../../../../x86_64-suse-linux/bin/ld: /usr/local/lib/libgrpc.so.39.0.0: undefined reference to `re2::RE2::GlobalReplace(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >*, re2::RE2 const&, absl::lts_20240116::string_view)'
collect2: error: ld returned 1 exit status
```

Reply:
Ah, RE2 also uses Abseil, oh the joys.